
diff -u -r a/def.bzl b/def.bzl
--- a/def.bzl	2024-11-08 13:42:27.733022366 -0800
+++ b/def.bzl	2024-11-08 13:44:45.089900166 -0800
@@ -16,6 +16,7 @@
     "@bazel_skylib//lib:shell.bzl",
     "shell",
 )
+load("@rules_shell//shell:sh_binary.bzl", "sh_binary")
 load(
     "@bazel_gazelle_is_bazel_module//:defs.bzl",
     "GAZELLE_IS_BAZEL_MODULE",
@@ -185,7 +186,7 @@
         tags = tags,
         **kwargs
     )
-    native.sh_binary(
+    sh_binary(
         name = name,
         srcs = [runner_name],
         tags = tags,
diff -u -r a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel	2024-11-08 13:42:23.860997684 -0800
+++ b/MODULE.bazel	2024-11-08 13:43:46.961528172 -0800
@@ -8,6 +8,7 @@
 bazel_dep(name = "protobuf", version = "3.19.6", repo_name = "com_google_protobuf")
 bazel_dep(name = "rules_go", version = "0.41.0", repo_name = "io_bazel_rules_go")
 bazel_dep(name = "rules_proto", version = "4.0.0")
+bazel_dep(name = "rules_shell", version = "0.2.0")

 go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")

